src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -699,22 +702,21 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
         const { sharedLinkTracking, sharedLinkEmailTracking } = trackingProps;
         const { isEmailLinkSectionExpanded, isInviteSectionExpanded, showCollaboratorList } = this.state;
 
-        // Only show the restriction warning on the main page of the USM where the email and share link option is available
-        const showShareRestrictionWarning =
-            !isEmailLinkSectionExpanded &&
-            !isInviteSectionExpanded &&
-            !showCollaboratorList &&
-            allShareRestrictionWarning;
+        const isMainUSMSectionShown = !isEmailLinkSectionExpanded && !isInviteSectionExpanded && !showCollaboratorList;
```

Comment:
not sure if these are better names but what about: `isInitialStateShown` or `hasExpandedSections` (requires inverting the boolean logic)

"main USM section" sounds a little ambiguous. i.e. what is the main section?

Reply:
Thanks @tjuanitas . Actually I like the hasExpandedSections better so I updated the code